radius.js

```diff
@@ -0,0 +1,26 @@
+//finds a suitable radius given a set of coordinates
+//coords should be an array of coordinates of the form: (latitude, longitude)
+
+//computes the distance formula for all possible pairs of points
+//returns the maximum * 2
+
+var radius = function (coords) {
+  var matrix = [];
+  for (var i = 0 ; i<coords.length; i++) {
+    var row = [];
+    for (var j = 0; j<coords.length; j++) { 
+      row.push(Math.sqrt(Math.pow((coords[i][0] + coords[j][0]), 2) +
```

Comment:
I believe this function is incomplete as it doesn't seem to be able to handle negative coordinates. I also don't think it correctly finds the distance between two sets of positive coordinates unless I am misreading the formula.